src/index.html

```diff
@@ -9,8 +9,41 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <!-- <link rel="stylesheet" href="images/Search.svg"> -->
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     method="post"
+     class="forms"
+     data-qa="big"
+     >
+      <input class="big-element"
```

Comment:
If you use BEM methodology every element has to have a block name as a prefix.
And you can use a modifier if you want to use some individual styles.
```
class="form__input form__input--big"
```